tests/env/docker-compose-otel.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   collector:
-    image: otel/opentelemetry-collector-contrib
+    image: otel/opentelemetry-collector-contrib:0.110.0
```

Comment:
No need to stick on a specific version, I think we can just update the `otel-collector-config.yml`: https://github.com/roadrunner-server/grpc/commit/52f3bd79245e7be30905f384cd23010d206d2998

Reply:
Sorry, wrong link: https://github.com/roadrunner-server/grpc/blob/master/tests/env/otel-collector-config.yml